components/Hero/Hero.jsx

```diff
@@ -14,19 +15,21 @@ const Hero = ({
     imageFilename,
     ...other
 }) => {
-    let styles = {}
-    if(progressiveImage){
+    let backgroundPath = ''
+    if (progressiveImage) {
```

Comment:
Is this whole bit needed? cant we just put /images/hero/herobackground.jpg on line 28 or something

Reply:
I did this just in case we need it somewhere else but I can simplify it